snapcraft/plugins/rust.py

```diff
@@ -54,6 +54,12 @@ def schema(cls):
 
     def __init__(self, name, options, project):
         super().__init__(name, options, project)
+        self.build_packages.extend([
+            'build-essential',
+            'git',
```

Comment:
Git should be pulled in simply by using a git source. It shouldn't be required here.